app/models/vita_partner.rb

```diff
@@ -31,6 +31,7 @@ class VitaPartner < ApplicationRecord
   has_many :intakes
   has_many :source_parameters
   has_many :users
+  has_many :vita_partner_zip_codes
```

Comment:
i wonder if we should change this to has_many :serviced_zip_codes, class_name: "VitaPartnerZipCode" so that it's a little more readable.

Reply:
Great idea!